core/src/main/java/com/scalar/db/api/DistributedTransactionManager.java

```diff
@@ -184,9 +196,12 @@ DistributedTransaction start(String txId, Isolation isolation, SerializableStrat
    * @param txId the transaction ID
    * @return {@link DistributedTransaction}
    * @throws TransactionNotFoundException if the transaction associated with the specified
-   *     transaction ID is not found
+   *     transaction ID is not found. You can retry the same transaction from the beginning in this
+   *     case
+   * @throws TransactionException if resuming the transaction fails
    */
-  DistributedTransaction resume(String txId) throws TransactionNotFoundException;
+  DistributedTransaction resume(String txId)
+      throws TransactionNotFoundException, TransactionException;
```

Comment:
Made `resume()` throw `TransactionNotFoundException` and `TransactionException`. Actually, it's incompatible change, but this `resume()` is not used widely, so I think it's okay.